ote/src/cljs/ote/views/transit_visualization/calendar.cljs

```diff
@@ -116,16 +116,17 @@
           :read :different-week-date
           :col-style style-base/table-col-style-wrap
           :format (fn [different-week-date]
-                    [:div
-                     [:span (stylefy/use-style {;; nowrap for the "3 pv" part to prevent breaking "pv" alone to new row.
-                                                :white-space "nowrap"})
-                      (str (time/days-until different-week-date) " " (tr [:common-texts :time-days-abbr]) " ")]
-                     [:span (stylefy/use-style {:color "gray"
-                                                :overflow-wrap "break-word"})
-                      (str "("
-                           (day-of-week-number->text (t/day-of-week (time/js-date->goog-date different-week-date)))
-                           " "
-                           (time/format-timestamp->date-for-ui different-week-date) ")")]])}
+                    (let [ddate (if different-week-date different-week-date (time/now))]
+                      [:div
```

Comment:
I guess this will do, but commenting that this styling seems similar to what is done in `ote.style.transit_changes.cljs` and referred if transit_changes.cljs and transit_visualization.cljs.
E.g. `transit-changes-icon-row-container`. 
Calendar row style could be defined there as well, if you have the energy.

Reply:
This is copy paste from transit_changes route list with minor changes. I do not want to refactor this now. Maybe when we add the change status (or and) comments.